src/data_helper.py

```diff
@@ -264,7 +285,14 @@ def _val_transform_to_matrices(self, *args):
         # Augment the image `img` here
 
         # Convert to RGB and normalize
-        img_array = np.array(img.convert("RGB"), dtype=np.float32) / 255
+        img_array = np.array(img.convert("RGB"), dtype=np.float32)
+        img_array = img_array[:, :, ::-1]
+        # Zero-center by mean pixel
+        img_array[:, :, 0] -= 103.939
+        img_array[:, :, 1] -= 116.779
+        img_array[:, :, 2] -= 123.68
+        img_array = img_array / 255
```

Comment:
We are dividing by 255 in the training set if you check the `ImageDataGenerator` we use `rescale=1./255`. So in this way actually training/validation/predicitons all consistently divide by 255.